### PR TITLE
Pretty print plus infer

### PR DESCRIPTION
**Pull request by brian-prigge**
Adds pretty printing plus inference support to command line.

### CODE REVIEW

**Comment by brian-prigge** [comment]
#5 #4

File: parserx.py [hunk 1]
```diff
@@ -79,7 +99,16 @@ def generate_output(n):
             print("Invalid. Enter input and output file names separated by a space.")
         except FileNotFoundError:
             print("Input file not found. Please try again.")
-
+    elif n == 3:
+        results = {}
+        results['parsed'] = SigParser().parse(" ".join(sys.argv[3:]))
+        results['inferred'] = SigParser().infer(results['parsed'], ndc=sys.argv[2]) 
+        print(json.dumps(results, indent=4))
+    elif n == 3:
```
Comment (jrlegrand) [inline parserx.py:107]: Please change this to n == `4`

I think this is just a typo.

**Review by jrlegrand — CHANGES_REQUESTED** [review]
Please see my one comment. Otherwise looks great! Thank you!

File: parserx.py (hunk 1 above)
Comment (brian-prigge) [reply parserx.py]: Good catch!

**Review by jrlegrand — APPROVED** [review]
Looks great! Thank you!

**Pull request merged**